### ISSUE TITLE
 jQuery.Deferred exception: Property "rulers"

### ISSUE THREAD

**Issue by EndersonPro** [ka215/jquery.timeline#44]
> jQuery.Deferred exception: Property "rulers" cannot set because undefined or invalid variable. TypeError: Property "rulers" cannot set because undefined or invalid variable.

![image](https://user-images.githubusercontent.com/29903127/62169591-356ff100-b2ee-11e9-82f0-6b49f6943795.png)

**Comment by zedle** [comment]
+1 getting this as well

**Comment by zedle** [comment]
figured this out, gotta set the rulers option in the init. not sure why this isn't in the docs.

**Comment by EndersonPro** [comment]
Could you show me how you solved it?

**Comment by ka215** [comment]
Hi there, and thank you for telling me.

It was a bug since version 2.0.0 beta 2.
Sorry I'd not noticed that. I'll fix the bug asap.

Thank you,

**Comment by FHantke** [comment]
@EndersonPro I do it like this:
```
<script>
    $(function () {
        let options = {
            ruler: {
                truncateLowers: false,
                top: {
                    lines: [ "year", "month", "weekday", "day" ],
                    height: 26,
                    fontSize: 12,
                    color: "#333",
                    background: "transparent",
                    locale: "de-DE",
                    format: {timeZone: "UTC", weekday: "short", year: "numeric", month: "long", day: "numeric"}
                },
                bottom: {
                    lines: [ "week", "year" ],
                }
            },
        }
        $("#myTimeline").Timeline(options);
    })
    </script>
```

**Comment by rkorzeniewski** [comment]
@ka215 did you fixed the bug?

**Comment by ka215** [comment]
Hi there,

I fixed that bug then released version 2.0.0 beta-3.

Thank you,

**Comment by ka215** [comment]
already fixed this.
